bin/deploy

```diff
@@ -11,14 +11,10 @@ bin/console rabbitmq-supervisor:control stop
 
 # Run composer install to make sure all non-dev dependencies are installed and parameters are defined
 composer install --no-dev --optimize-autoloader
-echo
 
-# Dump assetic assets
-bin/console assetic:dump
-
-# Dump FOSJsRouting routes
-echo "Dumping FOSJsRouting routes..."
-bin/console fos:js-routing:dump --target=web/js/routing
```

Comment:
We ARE still using FOSJsRouting...

Reply:
Ok, added it back. I also confirmed running this overwrites the previous routing file, so there's no need to clear it first.